versioned_docs/version-8.2/components/connectors/out-of-the-box-connectors/kafka-inbound.md

```diff
@@ -0,0 +1,121 @@
+---
+id: kafka-inbound
+title: Kafka Consumer Connector
+sidebar_label: Kafka Consumer Connector
+description: "The Kafka Consumer Connector allows you to connect your BPMN service with Kafka. Learn how to create a Kafka Consumer Connector and make it executable."
+---
+
+The **Kafka Consumer Connector** allows you to connect your BPMN service with [Kafka](https://kafka.apache.org/).
+
+## Prerequisites
+
+To use the **Kafka Consumer Connector**, you need to have a Kafka instance with configured bootstrap server.
+It is highly recommended not to expose your sensitive data as plain text, but rather use Camunda secrets. Follow our documentation on [managing secrets](/components/console/manage-clusters/manage-secrets.md) to learn more.
+
+## Create a Kafka Consumer Connector task
+
+To use the **Kafka Consumer Connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu icon, or create a new Connector task by using the **Append Connector** context menu. Follow our [guide on using Connectors](/components/connectors/use-connectors/index.md) to learn more.
```

Comment:
I deleted this part since it does not really make sense for inbound connectors. We cannot "create a new Connector task by using the **Append Connector** context menu". I think the list of steps below this section is enough. WDYT?